src/args.rs

```diff
@@ -731,8 +731,10 @@ impl<'a> ArgMatches<'a> {
         let mut specs = vec![
             "path:fg:magenta".parse().unwrap(),
             "line:fg:green".parse().unwrap(),
+            "column:fg:yellow".parse().unwrap(),
```

Comment:
Please see #377, which asks for colors on column numbers. What's the use case? Similarly for separators.